client/src/components/SponsorsComps/HomePageSponsorsSection.jsx

```diff
@@ -0,0 +1,98 @@
+/* eslint-disable prettier/prettier */
+import React from "react";
+import HomeCommunitySponsors from "./HomeCommunitySponsors";
+import styled from 'styled-components';
+import {
+  Container,
+  BreakLine,
+} from '../GlobalStyledComponents/ReversimStyledComps'
+import { SponsorMiniPremium } from './SponserGeneralComps';
+
+// Styled-components section
+
+const SponserSectionContainter = styled(Container)`
+  margin-top: 80px;
+  margin-bottom: 100px;
+  flex-direction: column;
+  flex-wrap: nowrap;
+`;
+
+const HeadingAlinger = styled.div`
+  ${ ( { theme: { space } } ) => `
+    width: 100%;
+    display: flex;
+    align-items: baseline;
+    margin-bottom: ${space.xxl};
+  `}
+`;
+
+const Heading = styled.h2`
+  ${ ( { theme: { color, font } } ) => `
+        width: inherit;
+        position: relative;
+        z-index: 1;
+        text-align: center;
+        font-weight: ${font.weight_normal};
+        font-family: ${font.main};
+        font-size: ${font.size_h2};
+        color: ${color.heading_2};
+      `}
+`;
+
+const HomeSponsors = styled.div`
+  ${ ( { theme: { mq } } ) => `
+      max-width: 1200px;
```

Comment:
I dont think we need it. When removed it is spread to 100% and is a little wider.

Anyway `SponserSectionContainter` the parent here, which acts as the Container, already has a max-width and some padding, so the children should just fall-in to that, make more sense.